spring-cloud-aws-cloudmap/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>io.awspring.cloud</groupId>
+		<artifactId>spring-cloud-aws</artifactId>
+		<version>3.0.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>spring-cloud-aws-cloudmap</artifactId>
+	<name>Spring Cloud Cloud Map</name>
+	<description>Spring Cloud AWS Cloud Map</description>
+
+	<dependencies>
```

Comment:
These dependencies are not used in `spring-cloud-aws-cloudmap` module. Since they are required by the autoconfiguration that is triggered only if these dependencies are on the classpath, they should go to starter module and autoconfiguration should trigger only if each of the required dependency is on the classpath.

Reply:
Sorry can you please elaborate the change, I am not able to understand the recommendation here

Reply:
Module should have only dependencies used in this very particular module.

Starter should list all the dependencies that are used additionally in autoconfiguration.

Autoconfiguration should be have `@ConditionalOnClass` listing classes from each dependency that is required for the integration to work.